### ISSUE TITLE
quantized Normal and Uniform Int/Float have different quantization points 

### ISSUE THREAD

**Issue by eddiebergman** [automl/ConfigSpace#264]
```python
from ConfigSpace import NormalIntegerHyperparameter, UniformIntegerHyperparameter

# Same bounds and q
u = UniformIntegerHyperparameter("u", 1, 10, q=3)
n = NormalIntegerHyperparameter("n", mu=5, sigma=2, q=3, lower=1, upper=10)

set([n.sample(rs) for _ in range(100)])
# {0, 3, 6, 9}

set([u.sample(rs) for _ in range(100)])
# {1, 4, 7, 10}
```

Given the same boundaries, I imagine the quantization points would be equal

**Comment by mfeurer** [comment]
That's a great finding. I think this should be handled as part or after #252.

**Comment by eddiebergman** [comment]
Closed by #346 which removes quantization
